cdap-app-fabric/src/main/java/io/cdap/cdap/app/store/Store.java

```diff
@@ -82,6 +84,10 @@ public interface Store {
   void setProvisioning(ProgramRunId id, Map<String, String> runtimeArgs,
                        Map<String, String> systemArgs, byte[] sourceId, ArtifactId artifactId);
 
+  void cleanupRepoSourceControlMeta(String namespace, HashSet<String> repoFiles);
```

Comment:
Dont use default store for these. We can directly use `SourceControlMetadataStore`s

Reply:
Done